src/PureCloud.Client/Contracts/IWorkforceManagementApi.cs

```diff
@@ -0,0 +1,1198 @@
+    Task<HrisTimeOffTypesResponse> CreateIntegrationsHriTimeofftypesJobsAsync(HrisTimeOffTypesRequest body, CancellationToken cancellationToken = default);
+    Task<AgentsIntegrationsListing> CreateAgentsIntegrationsHrisQueryAsync(QueryAgentsIntegrationsRequest body, CancellationToken cancellationToken = default);
+    Task<BulkUpdateAgentAlternativeShiftResponse> CreateAgentsMePossibleworkshiftsAsync(QueryAlternativeShiftForAgentsRequest body, CancellationToken cancellationToken = default);
+    Task<BuCurrentAgentScheduleSearchResponse> CreateAgentschedulesMineAsync(BuCurrentAgentScheduleSearchRequest body, CancellationToken cancellationToken = default);
+
+    Task<HrisIntegrationResponse> UpdateAgentIntegrationsHrisAsync(string agentId, AgentIntegrationsRequest body, CancellationToken cancellationToken = default);
+    Task<BuTimeOffLimitResponse> UpdateBusinessUnitTimeofflimitValuesAsync(string businessUnitId, string timeOffLimitId, BuSetTimeOffLimitValuesRequest body, CancellationToken cancellationToken = default);
+    Task<TimeOffLimit> UpdateManagementunitTimeofflimitValuesAsync(string managementUnitId, string timeOffLimitId, SetTimeOffLimitValuesRequest body, CancellationToken cancellationToken = default);
+
+    // Historical Data Operations
+    Task<HistoricalImportDeleteJobResponse> GetHistoricaldataBulkRemoveJobAsync(string jobId, CancellationToken cancellationToken = default);
+    Task<HistoricalImportDeleteJobListing> GetHistoricaldataBulkRemoveJobsAsync(CancellationToken cancellationToken = default);
+    Task<HistoricalImportStatusListing> GetHistoricaldataDeletejobAsync(CancellationToken cancellationToken = default);
+    Task<ImportStatusResponse> GetHistoricaldataImportstatusAsync(CancellationToken cancellationToken = default);
+    Task<ImportStatusResponse> GetHistoricaldataImportstatusJobIdAsync(string jobId, CancellationToken cancellationToken = default);
+
+    Task<HistoricalImportDeleteJobResponse> CreateHistoricaldataBulkRemoveJobsAsync(HistoricalImportDeleteJobRequest body, CancellationToken cancellationToken = default);
+    Task<HistoricalImportStatusListing> CreateHistoricaldataDeletejobAsync(CancellationToken cancellationToken = default);
+
+    // Notification and Schedule Operations
+    Task<UpdateNotificationsResponse> CreateNotificationsUpdateAsync(UpdateNotificationsRequest body, CancellationToken cancellationToken = default);
+    Task<UserScheduleContainer> CreateSchedulesAsync(CurrentUserScheduleRequestBody body, CancellationToken cancellationToken = default);
+    Task<WfmHistoricalAdherenceResponse> CreateTeamAdherenceHistoricalAsync(WfmHistoricalAdherenceQuery body, CancellationToken cancellationToken = default);
+    Task<WfmHistoricalShrinkageResponse> CreateTeamShrinkageJobsAsync(WfmHistoricalShrinkageRequest body, CancellationToken cancellationToken = default);
```

Comment:
Not all the methods on the interface are documented. Be sure to document any method in the interface. Also, be sure to add a new line after every method signature.  Additionally, at the end of the line you should only have a single new line.